template/web/wp-content/themes/<%= slug %>/app/Repositories/TermRepository.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Repository entity for retrieving post type objects.
+ *
+ * @package Studiometa
+ */
+
+namespace Studiometa\Repositories;
+
+use Timber\TermGetter;
+
+/** Class */
+final class TermRepository extends Repository {
+	const CLASS_TYPE = '\Timber\Term';
+
+	/**
+	 * Set what the Query will return
+	 *
+	 * @param array  $params     Query params.
+	 * @param string $class_type Class type for the query to return.
+	 * @return Timber\Term
+	 */
+	public function do_query( $params, $class_type ) {
```

Comment:
issue: remove the unused `$class_type` param

Reply:
@perruche @titouanmathis I think that we need to keep this param in case we want to use WordPress Term class. We need to specify a default param to '\Timber\Term' IMHO!

Reply:
The `$class_type` is stored in `static::CLASS_TYPE`, the param does not seem to be useful anymore.